contrib/edu-examples/tensorflow/cifar10_vgg16_tf_cpu.yaml

```diff
@@ -0,0 +1,32 @@
+protocolVersion: 2
+name: cifar10_vgg16_tf2_cpu_example
+type: job
+jobRetryCount: 0
+prerequisites:
+  - type: dockerimage
+    uri: 'openpai/standard:python_3.6-tensorflow_2.1.0-cpu'
+    name: docker_image_0
+taskRoles:
+  taskrole:
+    instances: 1
+    completion:
+      minFailedInstances: 1
+      minSucceededInstances: -1
+    taskRetryCount: 0
+    dockerImage: docker_image_0
+    resourcePerInstance:
+      gpu: 0
+      cpu: 4
```

Comment:
This should be `cpu: 1`? Since you mention only a single CPU is used before.

Reply:
In fact 4 cpu are used. Fixed in README.